$NetBSD: patch-docs_ld.lld.1,v 1.6 2019/11/03 12:40:40 kamil Exp $

Cherry-pick upstream patch commit 2a0fcae3d4d1fd85d6ae8378d7c6f12430c0087d
[lld] [ELF] Add '-z nognustack' opt to suppress emitting PT_GNU_STACK

--- docs/ld.lld.1.orig	2019-11-03 11:37:33.105878671 +0000
+++ docs/ld.lld.1
@@ -612,6 +612,11 @@ Set the
 .Dv DF_1_NOOPEN
 flag to indicate that the object may not be opened by
 .Xr dlopen 3 .
+.It Cm nognustack
+Do not emit the
+.Dv PT_GNU_STACK
+segment.
+.Pp
 .It Cm norelro
 Do not indicate that portions of the object shold be mapped read-only
 after initial relocation processing.
